packages/x-utils/tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "compilerOptions": {
+    "target": "es2019",
+    "module": "commonjs",
+    "outDir": "./dist/cjs",
+    "declaration": true,
+    "moduleResolution": "node",
+    "strict": true,
```

Comment:
I don't see this property being configured here. Am I missing something? We need it to be set to true in order to benefit from importing helpers from tslib instead of having to inject them in every single file where they're needed.